pkg/ir/common.go

```diff
@@ -27,5 +27,5 @@ type Writer interface {
 }
 
 type Collection interface {
-	Write(Writer) error
+	Write(Writer, string) error
```

Comment:
```suggestion
	Write(w Writer, vpc string) error
```
make it clear in the interface that the string should be a vpc